Modules/_sqlite/module.c

```diff
@@ -404,6 +404,54 @@ pysqlite_error_name(int rc)
     return NULL;
 }
 
+static int
+add_sequence_constants(PyObject *module)
+{
+    PyObject *kwd;
+    const char *_keywords[] = {
```

Comment:
When I said "we need it in C", I meant, we need the list of keywords given by sqlite3 directly, not by hardcoding it in C. Hardcoding it in C doesn't help.